index.js

```diff
@@ -25,15 +25,57 @@ function fastifyCors (fastify, opts, next) {
     allowedHeaders: null,
     maxAge: null,
     preflight: true,
-    hideOptionsRoute: true
+    hideOptionsRoute: true,
+    strictPreflight: false
```

Comment:
I would flip this to `true` and bump the major. This is a substantial change already.

wdyt @jsumners?

Reply:
Sounds good to me.

Reply:
@codyzu ^^